src/test/java/com/zorranlabs/interview/espn/EspnSeleniumTest2.java

```diff
@@ -0,0 +1,42 @@
+package com.zorranlabs.interview.espn;
+
+import junit.framework.TestCase;
+import org.hamcrest.CoreMatchers;
+import org.junit.Assert;
+import org.openqa.selenium.*;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class EspnSeleniumTest2 extends TestCase {
+    private WebDriver driver;
+
+    public void setUp() throws Exception {
+        driver = new FirefoxDriver();
+    }
+
+    public void testSimple() throws Exception {
+        // WebDriver driver;
+        driver.get("http://espn.go.com");
+
+        // Find a link
+        WebElement nfl_scores_link = driver.findElement(By.cssSelector("a[href*='http://scores.espn.go.com/nfl/scoreboard']"));
```

Comment:
Did you consider implementing a Page Object pattern? In general what would that look like?